xml/System.Windows.Forms/SplitterPanel.xml

```diff
@@ -319,15 +284,8 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets which edge of the <see cref="T:System.Windows.Forms.SplitContainer" /> that the <see cref="T:System.Windows.Forms.SplitterPanel" /> is docked to. This property is not relevant to this class.</summary>
-        <value>One of the <see cref="T:System.Windows.Forms.DockStyle" /> values.</value>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This property is not relevant to this class.  
-  
- ]]></format>
-        </remarks>
+        <value>One of the enumeration values that specifies the position and manner in which a control is docked.</value>
```

Comment:
I don't find `position and manner` descriptive. Would it be better to say the same as we said in the summary - `which edge`?

Reply:
`<value>` is describing the meaning of the returned value. The returned value is a `DockStyle` instance, which is documented exactly like that:

https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.dockstyle?view=netcore-3.0

Reply:
```suggestion
        <value>One of the enumeration values that specifies which edge of the <see cref="T:System.Windows.Forms.SplitContainer" /> that the <see cref="T:System.Windows.Forms.SplitterPanel" /> is docked to.</value>
```

Reply:
We could review the enum summary too.